bcManager/bcConfig.py

```diff
@@ -3,23 +3,18 @@
 
 # ###############################################################################
 
-class config:
-    auth_token = None
-    top_level_group = None
-    search_count = 10
+class bcConfig:
+    # Search Settings
+    SEARCH_COUNT = 10
+    PLAYLIST = "private"
+    SORT_BY = "replay-date"
+    SORT_DIR = "desc"
+
+    ZONE_ADJ = "-04:00"
```

Comment:
This zone adjustment doesn't take into account daylight savings time. Part of the year has Eastern at EST `-5`.